msteams-platform/tabs/how-to/conversational-tabs.md

```diff
@@ -47,9 +47,12 @@ The following image shows the conversation panel:
 If the user starts a conversation, it's important to listen for the callback of that event to retrieve and save the **conversationId**:
 
 ```javascript
-microsoftTeams.conversations.onStartConversation = (conversationResponse) => {
-    // console.log(conversationReponse.conversationId)
-};
+⁠microsoftTeams.conversations.openConversation({
+    ...,
+    onCloseConversation: (_) => {
+        ⁠// console.log(conversationResponse)
```

Comment:
@ydogandjiev, Yuri would it be possible to make these API public.

Reply:
Sure, feel free to make the change. You just need to remove the @hidden tag from them in the microsoft-teams-libary-js repo.

Reply:
@Wajeed-msft! FYIP

Reply:
@eoinobrien! we have an update from Yuri to remove the @hidden tag. Could you please help us here.

Reply:
@ydogandjiev I was under the impression that it is the responsibility of the feature team to keep their code up to date.

Either way I have created a PR to unhide these features, however, due to the structure of the code, I'm unable to expose the interfaces `OpenConversationRequest` and `ConversationResponse` that I think would be especially useful to developers. These interfaces are not imported into the `index.ts` that is used by typedoc and I wanted to avoid making code changes beyond unhiding the docs.

Are you able to schedule some work so that this public feature can be documented correctly for developers?

Thanks

https://github.com/OfficeDev/microsoft-teams-library-js/pull/1235